### ISSUE TITLE
nonStatic

### ISSUE THREAD

**Issue by woodyrew** [yeojz/metalsmith-react-templates#8]
Could I suggest renaming `nonStatic` to `static` or `isStatic` and having an alias called `nonStatic` for backwards compatibility?  Negated boolean variable names can cause unnecessary confusion.
e.g.
```javascript
if (nonStatic === false) {...} 
\\ or
if (isStatic === true) {...} 
```

I'm happy to fork and do a pull request? Excellent plugin apart from that.

**Comment by yeojz** [comment]
@woodyrew good point. 
I agree... let's put it as `isStatic`.

PR definitely welcome :+1:

**Comment by woodyrew** [comment]
Added PR #9

**Comment by yeojz** [comment]
Closed via #9
